sle15/profiles/standard.profile

```diff
@@ -110,6 +110,7 @@ selections:
     - sshd_disable_root_login
     - sshd_disable_empty_passwords
     - sshd_set_keepalive
+    - sshd_set_max_sessions
```

Comment:
The appropriate value of `sshd_max_sessions` needs to be selected in both profiles.

Reply:
Resolved per e78d219